man/sd_notify.xml

```diff
@@ -202,6 +202,14 @@
         watchdog is enabled. </para></listitem>
       </varlistentry>
 
+      <varlistentry>
```

Comment:
btw, we don't use Signed-off-by: on systemd. That's a kernel thing. Please drop from the commit msg